### PR TITLE
Mainly Testing work to allow the tests to pass

### PR DESCRIPTION
**Pull request by kiejgordon**
### Issue tracking

- [OJ-XXXX](https://govukverify.atlassian.net/browse/OJ-XXXX)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: test/browser/pages/relying-party.js [hunk 1]
```diff
@@ -8,9 +8,8 @@ module.exports = class PlaywrightDevPage {
   }
 
   async goto() {
-    const websiteHost = process.env.WEBSITE_HOST || "http://localhost:5050";
```
Comment (merlinc) [inline test/browser/pages/relying-party.js:11]: The webservice is set to run on port `5050` by default, we should stick to that in our tests as well

https://github.com/alphagov/di-ipv-cri-toy-front/blob/main/src/lib/config.js#L13

File: test/docker/compose.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  mocks:
+    image: outofcoffee/imposter
+    volumes:
+      - "../imposter:/opt/imposter/config"
+    ports:
+      - "8010:8080"
+    networks:
+      - mynetwork
+
+  #  core-stub:
```
Comment (merlinc) [inline test/docker/compose.yml:12]: If we're not using the Core stub in local tests, let's remove it from our file

File: test/browser/support/setup.js [hunk 3]
```diff
@@ -41,19 +41,9 @@ Before(async function ({ pickle } = {}) {
   }
 
   const header = tag?.name.substring(10);
+  console.log(header);
```
Comment (merlinc) [inline test/browser/support/setup.js:44]: When you're done with the debugging, we can remove the `console.log`

File: local.Dockerfile [hunk 4]
```diff
@@ -9,8 +9,9 @@ RUN npm install
 
 COPY . ./
 
-RUN npm build
+RUN npm run build
 
+RUN apk --update --no-cache add curl
```
Comment (merlinc) [inline local.Dockerfile:14]: What are we using `curl` for within the `local.Dockerfile` ?

**Review by merlinc — CHANGES_REQUESTED** [review]

File: local.Dockerfile (hunk 4 above)
Comment (kiejgordon) [reply local.Dockerfile:14]: curl is used for the health check for the cucumber test container to ensure the web is up and running to then continue to run the tests as there was issues with the web not being up in time for the cucumber to run

**Comment by merlinc** [comment]
These changes have been incorporated into other PRs, #15 #16 #35

**Pull request closed without merging**